packages/workflow/src/Interfaces.ts

```diff
@@ -1180,6 +1180,7 @@ export interface INodeTypeDescription extends INodeTypeBaseDescription {
 	};
 	webhooks?: IWebhookDescription[];
 	translation?: { [key: string]: object };
+	communityNode?: boolean;
```

Comment:
should not add to this type unless this is set on BE. because it's used on BE and FE. Either create a new type on FE that extends this one or update BE to return this flag.